### ISSUE TITLE
Module build failed

### ISSUE THREAD

**Issue by wedranb** [glennsl/bs-json#31]
Hi, I installed version `1.3.0` of this package and got this error during a compile time.

Module build failed: Error: We've found a bug for you!
`/../node_modules/@glennsl/bs-json/src/Json_decode.ml`

```bash
  The implementation src/Json_decode.mlast
       does not match the interface src/json_decode.cmi:
       Values do not match:
         val bool : Js.Json.t -> Js.boolean
       is not included in
         val bool : bool decoder
       File "./.../node_modules/@glennsl/bs-json/src/Json_decode.ml", line 18, characters 4-8:
    at <anonymous>claration
```

**Comment by glennsl** [comment]
Your `bs-platform` is out of date. Upgrade to 3.0.0, then it should work. You should have gotten a warning about this version mismtach too. `bs-json` 1.3.0 requries `bs-platform` 3.0.0.

**Comment by glennsl** [comment]
On the other hand, a minor release still shouldn't break your code like this... I'll see about makign a patch release that maintains backwards compatibility.

**Comment by glennsl** [comment]
I've released 1.3.1, which should be backwards comaptible with bs-platform 2.x. Thanks for reporting! 

(You should still upgrade bs-platform though :slightly_smiling_face:)
